### ISSUE TITLE
Suggesting new child term of "Abnormality of the lumbar spine"

### ISSUE THREAD

**Issue by theoriste** [obophenotype/human-phenotype-ontology#3682]
Suggest creating new child term of Abnormality of the lumbar spine [HP:0100712] 
New term label: Abnormality of the conus terminalis
New term definition: Abnormality in the tapered, lower end of the spinal cord.
New term comment (if any):
New term synonyms(if any): Abnormality of the conus medullaris
New term lay person synonyms (if any):
Reference (e.g., PubMed ID):     10903684; 26416933
Your biocurator ID for nanoattribution (if desired):

**Comment by pnrobinson** [comment]
I agree with this suggestion. One thing is that we are in the process of separating morphology and function abnormalities and so I think we should use the label Abnormal conus terminalis morphology.

**Comment by pnrobinson** [comment]
I think this would probably be better as a child of  Abnormality of the spinal cord HP:0002143 
Note that there is already a term, "Tethered cord" https://hpo.jax.org/app/browse/term/HP:0002144, that could be a child of Abnormalit of the conus terminalis. I will also add a new term, Conus Medullaris Arteriovenous Malformation
